core/src/main/java/hudson/model/DirectoryBrowserSupport.java

```diff
@@ -520,7 +537,7 @@ private int dirRank(VirtualFile f) {
                                 break;
                             f = sub.get(0);
                             relPath += '/'+Util.rawEncode(f.getName());
-                            l.add(new Path(relPath,f.getName(),true,0, f.canRead()));
+                            l.add(new Path(relPath,f.getName(),true,0, f.canRead(),0));
```

Comment:
why 0?

Reply:
you are right, we could get it form the object, but it is not used.